jetty9/src/main/docker/jetty_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+JETTY_START_COMMAND="/scripts/jetty/jetty-start-command"
+
+if [ ! -f $JETTY_START_COMMAND ]; then
+  source $DIR/gen_start_command.sh $JETTY_START_COMMAND
```

Comment:
Agreed, we don't need to document `gen_start_command`. Just do it on startup.
@jmcc0nn3ll The reason we're doing all this is not for optimizing the start time, it's more to fix the issue with the Cloud Debugger which is caused by `start.jar` spinning up a 2nd JVM.